frontend/src/components/Header/components/AuthButtons/index.tsx

```diff
@@ -41,29 +37,50 @@ const AuthButtons = (): JSX.Element | null => {
 
 function LoggedInButtons({ name, email }: { name?: string; email?: string }) {
   const authName = isEmail(name) ? "Account" : name;
+  const [anchorEl, setAnchorEl] = useState<Element | null>(null);
+
+  const handleClick: MouseEventHandler<HTMLElement> = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
 
   return (
-    <Popover content={<Content />}>
-      <Button minimal rightIcon={IconNames.CHEVRON_DOWN} text={authName} />
-    </Popover>
+    <>
+      <Content />
+      <Button
+        onClick={handleClick}
+        minimal
+        rightIcon={IconNames.CHEVRON_DOWN}
+        text={authName}
+      />
+    </>
   );
 
   function Content() {
     const curatorAPIFeature = get(FEATURES.CURATOR_API);
     return (
-      <Menu>
+      <Menu
+        anchorEl={anchorEl}
+        keepMounted
+        open={Boolean(anchorEl)}
+        onClose={handleClose}
+        getContentAnchorEl={null}
+      >
+        <MenuItem disabled data-testid="user-email">
+          Logged in as: {email}
+        </MenuItem>
         {curatorAPIFeature && (
-          <>
+          <div>
             <CuratorAPIKeyGenerator />
             <MenuDivider />
-          </>
+          </div>
```

Comment:
why the div?

Reply:
MUI Menu warns that it doesn't support fragments! Not sure why such limitation though